src/background/tab-manager.ts

```diff
@@ -79,6 +79,23 @@ export default class TabManager {
                 a.download = name;
                 a.click();
             }
+            if (type === 'export-css-proxy') {
+                const tabs = await queryTabs({});
+                const activeURL = await this.getActiveTabURL();
+                tabs.filter((tab) => this.ports.has(tab.id))
```

Comment:
This whole tabs is having a bug that it somehow send multiple messages that is equal to the amount of tabs you have open.